fetch-utils.js

```diff
@@ -61,10 +61,10 @@ export const createDefaultCity = async() => {
     return checkError(newCity);
 };
 
-export const updateImage = async(targetedElement, value) => {
+export const updateImage = async(key, value) => {
     await client    
         .from('cities')
-        .update({ [targetedElement] : value })
+        .update({ [key] : value })
         .match({ user_id: client.auth.user().id });
     render(await getCity());
 };
```

Comment:
Nice work with this updateImage function!  It is always nice to abstract some functions that are doing very similar work into a more broadly applicable utility function like this.